src/app/organizations/organization-details/manage-projects/create-new-project/create-new-project.component.ts

```diff
@@ -76,6 +76,10 @@ export class CreateNewProjectComponent {
             dateInputFormat: 'DD-MM-YYYY'
         });
     }
+
+    backToOrganizationDetailsScreen() {
+        this.router.navigate([`/dash/manage_groups/${this.orgID}`]);
```

Comment:
Can you try something like [this](https://stackoverflow.com/a/53327657)?
It might be unnecessary to hold orgID in a variable if we don't need it. Just try how it works.